### ISSUE TITLE
Hexagon can't coexist with Start

### ISSUE THREAD

**Issue by daviddeutsch** [aren227/TheWitness-LockScreen#31]
Kind of the same issue as https://github.com/thefifthmatt/windmill-client/issues/28 but it actually works with the End marker, just not the Start marker

**Comment by aren227** [comment]
That's because start marker is a rule assigned to the vertex. Hexagon is also a rule can be assigned to the vertex. But the problem is currently each vertex can only have one rule. So they can't coexist.

Maybe I should change the code so that each vertex can have more than one rule.

**Comment by daviddeutsch** [comment]
Yeah, in this case, I think that's justified because it's literally a feature that is used in a puzzle in the game.
